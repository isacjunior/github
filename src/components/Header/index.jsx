import React from 'react'
import HeaderStyled from './styled'

const Hero = () => (
  <HeaderStyled>
    <div className="container">
      <h1>Github</h1> 
      <p>Lorem Ipsum is placeholder text commonly used in the graphic, print, and publishing industries for previewing layouts and visual mockups.</p> 
    </div>
  </HeaderStyled>
)

export default Hero