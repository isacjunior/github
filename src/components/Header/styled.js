import styled from 'styled-components'

const HeaderStyled = styled.div`
  background: #556271;
  padding: 50px 0 80px 0;
  color: #fff;
  font-size: 18px;

  h1 {
    color: #FF6A67;
    font-size: 48px;
  }

  .icon {
    font-size: 50px;
    display: inline-block
  }

`

export default HeaderStyled