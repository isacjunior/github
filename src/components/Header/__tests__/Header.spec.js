import React from 'react'
import { mount } from 'enzyme'
import Header from '../index'

describe('Header', () => {
  it('Header render', () => {
    mount(<Header />)
  })

  it('Snapshot Header', () => {
    const wrapper = mount(<Header />)
    expect(wrapper).toMatchSnapshot
  })
})