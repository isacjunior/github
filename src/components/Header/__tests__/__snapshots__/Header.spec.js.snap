// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Snapshot Header 1`] = `
<Hero>
  <styled.div>
    <div
      className="sc-bdVaJa kAOJZq"
    >
      <div
        className="container"
      >
        <div
          className="row"
        >
          <div
            className="col-md-12"
          >
            <div
              className="icon"
            >
              <Icon
                icon="fa-github"
              >
                <i
                  className="fa fa-github"
                />
              </Icon>
            </div>
            <h1>
              Github
            </h1>
          </div>
        </div>
        <div
          className="row"
        >
          <div
            className="col-md-6"
          >
            <p>
              Lorem Ipsum is placeholder text commonly used in the graphic, print, and publishing industries for previewing layouts and visual mockups.
            </p>
          </div>
        </div>
      </div>
    </div>
  </styled.div>
</Hero>
`;
